core/tests/protractor_utils/AdminPage.js

```diff
@@ -110,12 +110,18 @@ var AdminPage = function() {
     this.reloadExploration = async function(name) {
       await this.get();
       explorationElements.map(async function(explorationElement) {
-        var title = await getExplorationTitleElement(explorationElement)
-          .getText();
+        await waitFor.visibilityOf(
+          getExplorationTitleElement(explorationElement));
+        var title = await getExplorationTitleElement(
+          explorationElement).getText();
 
         // We use match here in case there is whitespace around the name.
         if (title.match(name)) {
-          await getExplorationElementReloadButton(explorationElement).click();
+          var explorationElementReloadButton =
```

Comment:
Please break after `(` here (see [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#javascript)).

Reply:
but it exceeds the line limit and gives me error in lint checks